translate/storage/fpo.py

```diff
@@ -31,6 +31,7 @@
 import copy
 import re
 import six
+import logging
```

Comment:
This import must be two lines up. Imports must be alphabetically arranged. See our styleguide on imports.